lib/drivers/dexcom/dexcomDriver.js

```diff
@@ -142,70 +168,73 @@ module.exports = function (config) {
     DANISH: { value: 0x0406, name: 'Danish' },
     HUNGARIAN: { value: 0x040E, name: 'Hungarian' },
     NORWEGIAN_BOKMAL: { value: 0x0414, name: 'Norwegian (Bokmål)' },
-    TURKISH: { value: 0x041F, name: 'Turkish' }
+    TURKISH: { value: 0x041F, name: 'Turkish' },
   };
 
-  var ALARM_PROFILES = [
+  const ALARM_PROFILES = [
     'Vibrate',
     'Soft',
     'Normal',
     'Attentive',
-    'Hyposafe'
+    'Hyposafe',
   ];
 
-  var hasFlag = function (flag, v) {
+  const hasFlag = (flag, v) => {
```

Comment:
General style query: why are these named arrow functions, as opposed to regular functions?

Reply:
eslint flagged these as they're unnamed functions: https://eslint.org/docs/rules/func-names. I've been changing everything to arrow functions because I think it looks cleaner and it matches the rest of the driver styles.